docs/source/resources-algorithms.rst

```diff
@@ -3,4 +3,387 @@ OpenDCS Algorithms - Resources
 ###################################
 
 
-Content coming soon ...
+****************************
+Algorithms Resources - Recap
+****************************
+
+There are a number of "Standard" or "Baseline" algorithms that come 
+with OpenDCS when it is installed.  These can be thought of as banked 
+algorithms or functions that can be called upon when setting up a computation.
+These standard algorithms are generally basic arithmetic or time series
+manipulation or transformations.  For example, the average algorithm
+(AverageAlgorithm)can be used to create a daily or monthly time series 
+from an hourly or daily time series, respectively.  In addition to 
+simple arithmetic, or transformations, there are some set defined 
+hydrologic specific algorithms that are commonly used in water management.
```

Comment:
Would `water resources` be better here over `water management`?

Reply:
I am open to any of the following: 
- water resources
- water resources management
- water management

Reply:
Yeah, they all fit, but which is sufficient generic. Let's ask more of the group.

@opendcs/opendcs-triage @opendcs/opendcs-core-devs @dsvcw

Thoughts anyone?